pages/_document.tsx

```diff
@@ -9,6 +9,22 @@ export default function Document() {
           src="https://identity.netlify.com/v1/netlify-identity-widget.js"
         />
         <link rel="shortcut icon" href="/favicon.ico" />
+        <link rel="manifest" href="/manifest.json" />
+        <link rel="apple-touch-icon" href="/icon.png" />
+        <meta name="application-name" content="Anarres.fm" />
```

Comment:
Curious, did you use a tool to generate these? Images and code. Did you use the svg as a source? Should we put a comment maybe, or something at the README so we can remember how to do it next time?

Reply:
Good point. Added a readme section ;)